src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -583,7 +585,7 @@ object ScalafixPlugin extends AutoPlugin {
               case "--stdout" =>
                 // --stdout cannot be cached as we don't capture the output to replay it
                 throw StampingImpossible
-              case "--tool-classpath" =>
+              case tcp if tcp.startsWith("--tool-classpath") =>
```

Comment:
this PR exposed that a scripted was passing for the wrong reason: a scalafix invocation was failing because of a cache miss recompilation while it should have failed in scalafix itself

https://github.com/scalacenter/sbt-scalafix/blob/9ceb2c5b0f21e802826d04c5696a80affc63c1d3/src/sbt-test/skip-windows/caching/test#L186-L193